core/src/main/java/io/lionweb/lioncore/java/serialization/NodePopulator.java

```diff
@@ -31,13 +31,17 @@ class NodePopulator {
   }
 
   void populateClassifierInstance(
-      ClassifierInstance<?> node, SerializedClassifierInstance serializedClassifierInstance) {
-    populateContainments(node, serializedClassifierInstance);
-    populateNodeReferences(node, serializedClassifierInstance);
+      ClassifierInstance<?> node,
+      SerializedClassifierInstance serializedClassifierInstance,
+      ClassifierInstanceResolver classifierInstanceResolver) {
```

Comment:
`classifierInstanceResolver` is used in every method in this class. Maybe turn it into a field?

Reply:
The funny thing is that we created the field, but then we forgot to actually use it... I will fix it